Task 4/Task4/Actions/Change.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Actions
+{
+    public class Change : FileAction
```

Comment:
I think these names are too vague. Probably, `ChangeFileAction` would be better - less confusion about purpose